CI_YAML.md

```diff
@@ -160,44 +283,210 @@ dependencies: >-
     {"dependency": "goldctl"}
   ]
 ```
+</td>
+  </tr>
+  <tr>
+    <td>device_type</td>
+    <td>The phone device type a target will be running against. For host only targets that do not need
+    a phone, a `none` value should be used.
+    <br>
+    Note: This property will be auto populated to CI builder dimensions, based on which CI determines which
+    testbed to run this target.
+</td>
+    <td>N/A</td>
+    <td>string</td>
+    <td>
 
-**tags**: JSON list of strings. These are currently only used in flutter/flutter to help
-with TESTOWNERSHIP and test flakiness.
+``` yaml
+device_type: "msm8952"
+```
+</td>
+  </tr>
+  <tr>
+    <td>drone_dimensions</td>
+    <td>A list of testbed dimensions which the CI determines what testbed to assign a subbuild drone of a target to. This
+    property will be auto populated to CI dimensions of a subbuild triggered from the orchestrator target.
+</td>
+    <td>N/A</td>
+    <td>string</td>
+    <td>
 
-Example
-```yaml
-tags: >
-  ["devicelab","hostonly"]
+``` yaml
+drone_dimensions: >
+  ["device_type=none", "os=Linux"]
 ```
+</td>
+  </tr>
+  <tr>
+    <td>$flutter/osx_sdk</td>
+    <td>Xcode configs including sdk and runtime</td>
+    <td>N/A</td>
+    <td>map</td>
+    <td>
 
-**test_timeout_secs** String determining seconds before timeout for an individual test step.
-Note that this is the timeout for a single test step rather than the entire build execution
-timeout.
+``` yaml
+$flutter/osx_sdk : >-
+  {
+    "sdk_version": "14e222b",
+    "runtime_versions":
+      [
+        "ios-16-4_14e222b",
+        "ios-16-2_14c18"
+      ]
+  }
+```
+</td>
+  </tr>
+  <tr>
+    <td>gclient_variables</td>
+    <td>The gclient variables populated to recipes when checking out sources via gclient sync.
+    </td>
+    <td>N/A</td>
+    <td>map</td>
+    <td>
 
-Example
 ``` yaml
-test_timeout_secs: "2700"
+gclient_variables: >-
+  {
+    "download_android_deps": "true"
+  }
 ```
+</td>
+  </tr>
+  <tr>
+    <td>ignore_cache_paths</td>
+    <td>The paths of Engine checkout source that will be skipped when saved to CAS. Please reference to `cache_path`.
```

Comment:
I believe ignore_cache_paths paths are deleted. Not sure if that is meaningful here.

Reply:
It's still here? https://github.com/flutter/engine/blob/main/.ci.yaml#L166